arches_querysets/querysets.py

```diff
@@ -137,46 +136,9 @@ def _prefetch_related_objects(self):
                 if node.nodegroup_id == tile.nodegroup_id:
                     tile_val = getattr(tile, node.alias, NOT_PROVIDED)
                     if tile_val is not NOT_PROVIDED:
-                        datatype_instance = datatype_factory.get_instance(node.datatype)
```

Comment:
This is where we attach tile information to SemanticTile python instances. This was getting way too fat, so I factored it out.